.github/workflows/pr.yml

```diff
@@ -397,9 +397,40 @@ jobs:
           python-version: ${{matrix.python}}
         if: matrix.test-suite == 'integration' || matrix.test-suite == 'notebook'
 
+      - uses: conda-incubator/setup-miniconda@v2
```

Comment:
YAML files are done and ready for review.
Added a new test matrix (testing with conda).